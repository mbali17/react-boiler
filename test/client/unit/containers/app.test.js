import React from 'react';
import { mount, shallow } from 'enzyme';

import App from '../../../../src/containers/app';;

describe('Test app.js', () => {
  let wrapper;

  afterEach(() => {
    if (wrapper) wrapper.unmount(wrapper);
  });

  test('Test core app render', (done) => {
    wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
    done();
  });
});
