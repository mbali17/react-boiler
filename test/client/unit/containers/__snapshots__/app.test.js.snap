// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app.js Test core app render 1`] = `
<form>
  <Connect(InputField)
    fieldName="name"
    key="0"
  />
  <Connect(InputField)
    fieldName="email"
    key="1"
  />
  <Connect(InputField)
    fieldName="company"
    key="2"
  />
  <Connect(InputField)
    fieldName="phone"
    key="3"
  />
  <Connect(SubmitButton) />
</form>
`;
